SlevomatCodingStandard/Sniffs/Commenting/EmptyFunctionBodyCommentSniff.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\FunctionHelper;
+use const T_FUNCTION;
+use const T_WHITESPACE;
+
+class EmptyFunctionBodyCommentSniff implements Sniff
+{
+
+	public const CODE_FUNCTION_MISSING_COMMENT = 'FunctionMissingComment';
+	public const CODE_METHOD_MISSING_COMMENT = 'MethodMissingComment';
```

Comment:
Use just one code please: `CODE_EMPTY_FUNCTION`

Reply:
Sure I can... I just have a condition there, which add different code for class method and normal function. If I will have only 1 code, then that code should be also removed. Right?

Reply:
Yes :)